index.coffee

```diff
@@ -364,6 +364,10 @@ module.exports = class Flatten
         return results unless normalizedChildren.length
         if tag.attribs?.href
           normalizedChildren[0].html = @tagToHtml tag, id
+        if tag.attribs?['data-role'] is 'cta' and tag.attribs.href
+          normalizedChildren[0].type = 'cta'
+          normalizedChildren[0].url = tag.attribs.href
+          normalizedChildren[0].label = @tagToText(tag)
```

Comment:
We need support for linking to a Reserve CTA from attributes, like in https://github.com/the-grid/apidocs/blob/master/schemata/sitepurpose.yaml#L22